### ISSUE TITLE
the inconsistency between benchmark::evm::TestCircuit and zkevm::evm::TestCircuit

### ISSUE THREAD

**Issue by lispc** [privacy-scaling-explorations/zkevm-circuits#458]
In zkevm_circuits, the TestCircuit of evm circuit contains 31 columns of randomness, while in benchmark, evm::TestCircuit we use 31 constants

<https://github.com/appliedzkp/zkevm-circuits/blob/29ccd5f656fed8e3f8c436ab944acad742c63cf0/circuit-benchmarks/src/evm_circuit.rs#L30>

So all the benchmark results may not indicate very accurate perf result of real evm citcuit ( Degree of circuit in benchmark is 1 lower, column VS constant).    So we change randomess in benchmark::TestCircuit to use 31 columns too?

**Comment by han0110** [comment]
The one in benchmark (using `Expression::Constant`) is the behavior we would like (no extra degree). The reason testing is using instance columns is a mistake (and forgot to be updated), I thought we could simulate extra challenge by just using instance column without any change to halo2, but it turns out to be too tricky.

We should just use `Expression::Constant` also for testing, then wait for extra challenge is supported and replace it with `Expression::Challenge` (a prototype has been implemented, but still trying to minimize the changes).

**Comment by lispc** [comment]
“We should just use Expression::Constant also for testing" will be a small PR. But will it worth to change now? considering we finally will replace it with Expression::Challenge after the halo2 prototype merged? 

It seems the inconsistency does not block anybody's work now?

**Comment by CPerezz** [comment]
No it doesn't. But this might affect the results that we get for our circuit benchmarks. 

IDK if significantlly enough to adress it now. WDYT @han0110 ?

**Comment by han0110** [comment]
I think the benchmark is running the one we expect (no extra degree), so the result should also be expected. I also don't think anything blocked by this now.

**Comment by CPerezz** [comment]
Adding `wontfix` since it's not a prio at all and therefore will not be addressed in the near future.

**Comment by ed255** [comment]
The EVM Circuit now uses the challenge API so the 31 instance columns that simulated the powers of randomness are no longer there :)
